### PR TITLE
Add parent type (e.g. (Birth)) in in the HOI card in case items.

### PR DESCRIPTION
**Pull request by LotharII**
Add parent type (e.g. (Birth)) in in the HOI card in case items.
[INT-1407]

### Jira Story

- [Relationship Types in HOI case history needs to be displayed by parent names in CWS-CARES INT-1407](https://osi-cwds.atlassian.net/browse/INT-1407)

## Description
Add parent type (e.g. (Birth)) in in the HOI card on case items. 
## Tests
- [ x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [ x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x ] My code follows the code style of this project.
- [x ] I have ran all tests for the project.
- [x ] I have ran lint/rubocop check for the changed/new files.
- [x ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/selectors/screening/historyOfInvolvementSelectors.js [hunk 1]
```diff
@@ -33,17 +33,30 @@ const getCaseCountyAndStatus = (hoiCase) => {
   }
 }
 
+const getParentsNames = (hoiCase, relationshipTypes) => {
+  const parentTypeRegExp = new RegExp('\\(.*?\\)')
```
Comment (money) [inline app/javascript/selectors/screening/historyOfInvolvementSelectors.js]: Is the Regexp being used any where else?

File: app/javascript/selectors/screening/historyOfInvolvementSelectors.js [hunk 2]
```diff
@@ -33,17 +33,30 @@ const getCaseCountyAndStatus = (hoiCase) => {
...
+  return hoiCase.get('parents', List()).map((parent) => {
+    var fullName = nameFormatter(parent.toJS())
```
Comment (money) [inline app/javascript/selectors/screening/historyOfInvolvementSelectors.js]: consider using `let` instead of `var`.

Comment (NagaPradeepGajula-osi) [reply app/javascript/selectors/screening/historyOfInvolvementSelectors.js]: nice catch @money 👍

File: app/javascript/selectors/screening/historyOfInvolvementSelectors.js (hunk 1 above)
Comment (LotharII) [reply app/javascript/selectors/screening/historyOfInvolvementSelectors.js]: No, it's the only place.

File: app/javascript/selectors/screening/historyOfInvolvementSelectors.js (hunk 2 above)
Comment (LotharII) [reply app/javascript/selectors/screening/historyOfInvolvementSelectors.js]: @money Thanks, done.

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Pull request merged**